tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -251,14 +302,25 @@ describe('Lobby - P1 Perspective', () => {
 
   it('Shows when oppenent Readies/Unreadies', () => {
     cy.contains('[data-cy=opponent-indicator]', opponentOne.username);
-    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-back-card"]').should('exist');
+
+    // opponent : back-card visible, ready-card not visible
+    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-back-card"]').should('have.css', 'rotate', 'y 0deg');
+    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]').should('have.css', 'rotate', 'y -180deg');
     cy.readyOpponent();
-    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]', {timeOut: 10000}).should('exist');
-    cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-back-card"]').should('exist');
+    // oponent: ready-card visible with a delay of 1 second
+    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-back-card"]', {timeOut: 10000}).should('have.css', 'rotate', 'y 180deg');
+    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]', {timeOut: 10000}).should('have.css', 'rotate', 'y 0deg');
+    // me: back-card visible
+    cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-back-card"]').should('have.css', 'rotate', 'y 0deg');
+    cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-ready-card"]').should('have.css', 'rotate', 'y -180deg');
```

Comment:
Those timeouts are suspicious. Was the app really taking 10 seconds to reflect the update?
```suggestion
    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-back-card"]', {timeOut: 10000});
    cy.get('[data-cy=opponent-indicator]').find('[data-cy="lobby-ready-card"]', {timeOut: 10000});
    // me: back-card visible
    cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-back-card"]');
    cy.get('[data-cy=my-indicator]').find('[data-cy="lobby-ready-card"]');
```